packages/material/scss/textarea/_layout.scss

```diff
@@ -25,7 +25,7 @@
 
 
         .k-textarea .k-input {
-            padding-top: calc( #{$textarea-padding-y} + 14px );
+            padding-top: calc( #{$textarea-padding-y} + 20px );
```

Comment:
is this always 20px, or is it 1 line? if the latter, consider using `$line-height-em` or a similar unit of measurement.

Reply:
I've added `$input-line-height-em` and `$textarea-line-height-em` variables and fixed the calculation :)